contracts/BaseJumpRateModelV2.sol

```diff
@@ -103,6 +103,8 @@ abstract contract BaseJumpRateModelV2 is InterestRateModel {
             return 0;
         }
 
+        // At this point, borrows should always be greater than zero; the condition below ensures that (cash + borrows - reserves) should never be zero.
```

Comment:
I think this constraint is harder than we need and it could block almost any operation in the market. For example, if we don't reduce the reserve for a long time, or if the liquidation is low because we have a big demand of borrows, we'll have `cash < reserves` and therefore we won't accept new supplies.

Maybe `require(cash + borrows - reserves > 0, "BaseJumpRateModelV2 : division by zero error");` could be more precise, avoiding the division by zero but at the same time allowing supplies in periods with low liquidation.

Moreover, I would add the same require statement in WhitePaperInterestRateModel.sol

What do you think?

Reply:
Right, but can we skip these changes because we're using a solidity version higher than 0.8 and this can be handled by default?

Reply:
We'll finally remove the require statement because the division by zero will generate a revert (we are using solidity => 0.8.0), so we don't need it